### PR TITLE
fix(cta): fix CTA styling

### PR DESCRIPTION
**Pull request by marionnegp**
## What I did

1.

## Testing Instructions

1.

### CODE REVIEW

**Comment by marionnegp** [comment]
@bennypowers, I think I figured out why some properties (e.g., the secondary CTA's border on dark) weren't showing up as expected. `.dark` is getting added to the `<span>` but not to the :host. I'm not sure why though.

Screen Shot 2022-12-15 at 12 52 24 PM

Screen Shot 2022-12-15 at 2 51 57 PM

**Comment by bennypowers** [comment]
maybe we can move some of those styles to the after pseudoelement

**Comment by bennypowers** [comment]
#675 might fix this, or at least, it should help

File: elements/rh-cta/rh-cta.css [hunk 1]
```diff
@@ -4,19 +4,12 @@
   z-index: 0;
   align-items: center;
   max-width: max-content;
-  background-color: var(--rh-cta-background-color);
```
Comment (bennypowers) [inline elements/rh-cta/rh-cta.css:7]: Impressive, @marionnegp Great job

**Review by bennypowers — APPROVED** [review]
Well done!

Loquacious Gnats Talk Mercilessly

**Pull request merged**